charts/flink-job/templates/service.yaml

```
@@ -6,6 +6,7 @@ kind: Service
 metadata:
   name: {{ include "flink-job.fullname" $ }}-metrics
   labels:
+    app: {{ include "flink-job.flinkDeploymentName" $ }}
```

We should probably just use the fullname here. The `flink-job.flinkDeploymentName` helper is the truncated fullname, which is needed to make sure that the resource names doesn't get too large, which is not necessary here
```suggestion
    app: {{ include "flink-job.fullname" $ }}
```